components/TopBar.tsx

```diff
@@ -154,6 +155,12 @@ export default function TopBar(): React.ReactElement {
               <Flex>
                 <VerticalBar />
               </Flex>
+              <Flex direction="row">
+                <ScheduleModal />
+              </Flex>
```

Comment:
Top bar has enough entries now that it can take up too much space. I think adding a 5th breakpoint value to the hamburger menu and nav might work! (lines 121 and 131 in this file)

![image](https://user-images.githubusercontent.com/20251243/103160100-04c00f80-479f-11eb-9221-f24ca423a54e.png)

Reply:
Oh, thanks for noticing! Fixed in https://github.com/cusec/2021/pull/99/commits/25607aafb1b42bac553867fe99a26031ab9622c8